about.html

```diff
@@ -22,13 +25,15 @@
 That includes behaviour from other instances - we have no problem with stopping to federate with asshole instances, or suspend contact with single accounts.
 </p>
 
-<hr>
+<h2>Automated posts</h2>
+
+We try to keep the <a href="https://chaos.social/web/timelines/public/local">local timeline</a> of our instance and the Fediverse in general a place of dialogue and human interaction. Therefore you must configure all automated posts to be <strong>unlisted</strong>! This means they will be only visible on your personal timeline and for your followers. This refers to all kind of automated posts, regardless if they are bots or any kind of feed poster or Twitter "Retweets". In case of crossposts (for example with Twitter) they must be "unlisted", too, whenever they include broken mentiones or they are not themselve in an understandable context.
+
+<strong>Note on "Retweets":</strong>
 
-We try to keep the <a href="https://chaos.social/web/timelines/public/local">local timeline</a> of our instance a place of dialogue and interaction.
-Therefore we'd like to ask you to keep all <strong>automated posts unlisted</strong>. Unlisted posts can be seen by everybody but do not show up on the local timeline.
-This extends to bots, feed posters, Twitter "retweets", and Twitter crossposts with broken mentions.
+In Mastadon and the Fediverse "Boosts", which are similar to "Retweets" in Twitter, are handled a bit differently: They are always published as "unlisted", which means they won't appear neither on the <a href="https://chaos.social/web/timelines/public/local">local</a> nor on the <a href="https://chaos.social/web/timelines/public">federated timeline</a>. Therefore we don't tolerate "Retweets" that are not marked as "unlisted".
```

Comment:
I still think that "This means they will be only visible on your personal timeline and for your followers.", and the complete "note on retweets" has no place in the /about. That would belong in a user guide for Mastodon (sp.), not in the rules section of an instance. As I said: the longer this section becomes, the fewer people will read it.

Reply:
Yes, you are right. It would be better placed in some "FAQ" or user guide. I just expect others to run into this frequently, because a lot of people will be coming from Twitter. And with you having at least a place to reference to that explains this shortly, the less you will loose time explaining it to newbies.

Reply:
I also added an issue in the crossposter that I use to change the default behaviour (https://github.com/foozmeat/moa/issues/62). This will hopefully also have a relevant impact.